packages/contracts/package.json

```diff
@@ -12,6 +12,7 @@
     "lint": "run-p --aggregate-output lint:*",
     "lint:typescript": "tslint --project ./",
     "lint:solidity": "solium --dir contracts/",
+    "migrate": "truffle migrate",
```

Comment:
might want to rename this command 'migrate-mainnet' since it's a "pretty big deal". maybe add another command for 'migrate-rinkeby'?

Reply:
It actually throws an error on my computer if no network is specified. I wonder why it works differently on yours

Reply:
actually you're right. It requires us to specify network because we don't have a default configuration specified in `truffle-config.js`. So as long as we don't define a default configuration we're good.